DANS-KNAW/easy-deposit
# Implementation of "virtually valid" bags.

fixes EASY-1096
#### When applied it will
- download those files in fetch.txt that refer outside the bag store
- validate the bag
- if only 'missing payload files' validation errors, proceed
- if missing payload files are the same as the files in fetch.txt referring to the bag store, proceed
- verify the checksums of these files against the checksums in the referred bag
#### Where should the reviewer @DANS-KNAW/easy start?
#### How should this be manually tested?

See testing instructions in the project description. Notice: because at the moment there is no feedback about the uri of the newly created bag, you have to use the test bags in src/test/resources/simple-sequence.  Now you also have to copy the contents of the src/test/resources/bag-store directory to the /data/bag-store directory of the target server.
#### related pull requests on github

| repo | PR | note |
| --- | --- | --- |
| easy- | [PR#](PRlink) |  |
